make/testing_flags.mk

```diff
@@ -54,15 +54,12 @@ UNIT := $(GOLIST) | $(XARGS) env $(GOTEST) -tags="$(DEV_TAGS) $(LOG_TAGS)" $(TES
 UNIT_RACE := $(UNIT) -race
 endif
 
-
-# Construct the integration test command with the added build flags.
-ITEST_TAGS := $(DEV_TAGS) rpctest chainrpc walletrpc signrpc invoicesrpc autopilotrpc routerrpc
-
 # Default to btcd backend if not set.
-ifneq ($(backend),)
-ITEST_TAGS += ${backend}
-else
-ITEST_TAGS += btcd
+ifeq ($(backend),)
+backend = btcd
```

Comment:
Prior commit removes the btcd tag so this seems no longer needed?

Reply:
It is only needed for the `running integration tests with <backend> chain backend` logs